tox.ini

```diff
@@ -6,6 +6,7 @@ deps=
     flake8
     pytest
     sphinx-testing
+    .
```

Comment:
I think this is necessary to pull in the dependencies of this library (which in this PR includes jsonref) https://github.com/tk0miya/sphinxcontrib-jsonschema/pull/12/files/a8e2a38bed68fd58f93875446e64b7a2cdc513cc#diff-2eeaed663bd0d25b7e608891384b7298R7